src/core/ui/mainui.cpp

```diff
@@ -2,6 +2,8 @@
 
 #include "core/globals.hpp"
 
+#define INITIAL_ENGINE "TapeDeck"
```

Comment:
nitpick: prefer constexpr variables to macros

Reply:
Done.